tests/robustness/scenarios/scenarios.go

```diff
@@ -288,5 +291,22 @@ func Regression(t *testing.T) []TestScenario {
 			Cluster:   *e2e.NewConfig(opts...),
 		})
 	}
+
+	scenarios = append(scenarios, TestScenario{
+		Name:      "issue20693",
+		Profile:   traffic.HighTrafficProfile.WithoutCompaction().WithBackgroundWatchConfigInterval(10 * time.Millisecond).WithBackgroundWatchConfigRevisionOffset(-10),
+		Failpoint: failpoint.RaftAfterSaveSnapPanic,
+		Traffic:   traffic.Kubernetes,
+		Cluster: *e2e.NewConfig(
+			e2e.WithClusterSize(3),
+			e2e.WithCompactionBatchLimit(10),
+			e2e.WithSnapshotCount(50),
+			e2e.WithSnapshotCatchUpEntries(100),
+			e2e.WithGoFailEnabled(true),
+			e2e.WithPeerProxy(true),
+			e2e.WithIsPeerTLS(true),
```

Comment:
With the current root cause known, we can just disable most of those config changes.